draft.xml

```diff
@@ -101,500 +88,440 @@
-procedures.
-      </t>
-      <t>
-Some technical organizations operating networks that move from one conference
-location to the next have already experimentally published small geolocation
-feeds.  At least one consumer (Google) has incorporated these ad hoc feeds
-into a geolocation data pipeline.
-      </t>
-      <t>
-XXX NOTES TODO
-update geofeed contents (RIPE)
-6280 - 4.1.1 network-based positioning system
-
-implicit use policy for LRs
-andy newton's feedback (whois/rdap, dns, ...)
-consumer => Location Recipient
-
-privacy and other work into new #3 section about RM, LG, and LS
-then transmitted LR policy and LR section (current #4)
-
-A1-A4
-Look at PIDF-LO
-specify a media type (application/csv)
-allow repeated fields for multiple regions
-
-from London:
-- add use case for more granularity (?)
-- include ISP privacy info URL
-- User can discover if they're in a feed, increases transparency
-- RIPE whois field, RDAP
-- create link relationship, RFC 5988
-
-References:
-- opengeofeed.org
-- https://www.opengeofeed.org/feed/public.csv
-- Mozilla Location Service https://location.services.mozilla.com/
-- https://bugzilla.mozilla.org/show_bug.cgi?id=1042925
-      </t>
+      <t>This document records a format whereby a network operator can publish
+      a mapping of IP address prefixes to simplified geolocation information,
+      colloquially termed a geolocation "feed". Interested parties can poll
+      and parse these feeds to update or merge with other geolocation data
+      sources and procedures. This format intentioannly only has city level
+      granularity.</t>
+
+      <t>Some technical organizations operating networks that move from one
+      conference location to the next have already experimentally published
+      small geolocation feeds. At least one consumer (Google) has incorporated
+      these ad hoc feeds into a geolocation data pipeline, and is using it to
+      allow ISPs to inform them where the prefixes live. </t>
+
+      <t>[RFC Ed - Please remove publication: The IETF Meeting network
+      currently publishes a feed in this format at:
+      https://noc.ietf.org/geo/google.csv -- this has significantly cut down
+      on the number of "Gah! Why does the network believe I'm in Montreal,
+      that was last meeting! How the hell am I supposed to find a pub?!"
+      complaints. A number of other meeting networks, including RIPE and ICANN
+      publish this information as well, see below. ]</t>
+
+      <t>[ Ed note: Text inside square brackets ([]) is additional background
+      information, answers to frequently asked questions, general musings,
+      etc. They will be removed before publication.]</t>
+
+      <t>[ This document is being collaborated on in Github at:
+      https://github.com/google/self-published-geo . The most recent version
+      of the document, open issues, etc should all be available here. The
+      authors (gratefully) accept pull requests ]&lt;/t&gt;</t>
```

Comment:
Looks unintended. Remove?

Reply:
Actually, nope, was intended -- this is so that people who read the draft know where they can provide pull requests. It is between brackets, so the RFC Editor should remove before publication as RFC (but I'm happy to make it more explicit!)

Reply:
Whoops, I **completely** missed the ampersplat in the comment. Fixed in a5ec5efadd76c31a5a6b5e603b0f28289a5d0830